Misc/NEWS.d/next/Library/2021-04-16-17-32-44.bpo-43827.uJaXdP.rst

```diff
@@ -0,0 +1 @@
+Fix abc.ABCMeta conflicts with __init_subclass__
```

Comment:
This is not really enough context for someone to understand what is changing just from reading the changelog. Something more like this would be better:

"All positional-or-keyword parameters to `ABCMeta.__new__` are now positional-only to avoid conflicts with keyword arguments to be passed to `__init_subclass__`."

I am not super confident that that's the only thing that `ABCMeta`'s `**kwargs` is used for, so if it's used for other stuff, more general wording would be appropriate.